doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contribute to the terraform plugin.
+
+So you want to be a terraform-libvirt-plugin hacker.
+
```

Comment:
I don't think these 4 lines are needed. Since it's the CONTRIBUTING.md file, it's obvious that one wants to contribute.